test/e2e/e2e_suite_test.go

```diff
@@ -459,6 +483,51 @@ func dumpBootstrapClusterLogs(bootstrapClusterProxy framework.ClusterProxy) {
 	}
 }
 
+// configureIsolatedEnvironment sets the isolatedHostName variable to the IP of the first and only node in the boostrap cluster. Labels the node with
+// "ingress-ready" so that the nginx ingress controller can pick it up, required by kind. See: https://kind.sigs.k8s.io/docs/user/ingress/#create-cluster
```

Comment:
This ingress-ready label is only needed because of it being specified in the nginx ingress spec. I think it could be dropped for the use of existing `https://kubernetes.io/docs/reference/labels-annotations-taints/#node-role-kubernetes-io-control-plane` label, which is already present

Reply:
There is a node selector in Deployment that requires this label to be present.

Reply:
It also requires a kubeadm configuration to set these labels on the node, which we could not pass. But we have 1 control plane and so the destination is always known